gpkit/small_classes.py

```diff
@@ -93,6 +93,19 @@ def to_united_array(self, unitless_keys=(), united=False):
         "Converts all lists into array, potentially grabbing units from keys."
         _enray_and_unit_dict(self, self, unitless_keys, united)
 
+    def classify(self, cls):
+        _classify(self, cls)
+
+
+def _classify(d_in, cls):
+    "Converts all contained dictionaries into the given class."
+    for k, v in d_in.items():
+        if isinstance(v, dict) and v:
+            if isinstance(v.keys()[0], Strings):
+                _classify(v, cls)
+            else:
+                d_in[k] = cls(v)
```

Comment:
Nope, not anymore. Unless they replace instead of updating a substitution dict...